crates/nu-command/src/bytes/at.rs

```diff
@@ -159,3 +174,28 @@ fn action(input: &Value, args: &Arguments, head: Span) -> Value {
         ),
     }
 }
+
+fn resolve_relative_range(range: &Subbytes, len: &usize) -> (usize, usize) {
```

Comment:
No need to take a reference to a `usize`.
```suggestion
fn resolve_relative_range(range: &Subbytes, len: usize) -> (usize, usize) {
```

Reply:
Thanks! After looking up why, I can definitely see why it would be less efficient here to use a ref instead of copy! 😊